terraform/application/config/staging.tfvars.json

```diff
@@ -2,7 +2,7 @@
     "cluster": "test",
     "namespace": "bat-staging",
     "statuscake_alerts": {
-        "website_url": [ "https://staging.claim-funding-for-mentor-training.education.gov.uk/healthcheck.json" ],
+        "website_url": [ "https://staging.claim-funding-for-mentor-training.education.gov.uk/healthcheck/all" ],
```

Comment:
sandbox and staging didn't create the statuscake monitor previously as this requires "enable_monitoring" : true (which the prod tfvars does have). I'll also need to create the underlying monitor resource group when you add that (before you merge)

Reply:
As discussed out of channel we can handle the addition of "enable_monitoring" in a future PR, once the resource has been created. 👍